server/src/main/kotlin/org/javacs/kt/KotlinTextDocumentService.kt

```diff
@@ -312,6 +302,10 @@ class KotlinTextDocumentService(
     override fun close() {
         shutdownExecutors(awaitTermination = true)
     }
+
+    fun setTestLintRecompilationCallback(callback: () -> Unit) {
+        sp.beforeCompileCallback = callback;
+    }
```

Comment:
Properties should be used instead of raw getter/setter methods in Kotlin:

```kotlin
var testLintRecompilationCallback: () -> Unit
    set(callback) { sp.before.CompileCallback = callback }
```

Also, semicolons are not needed and should be omitted when possible.

Reply:
In general, we might want to find a more generic name for this property, maybe just `lineRecompilationCallback`.

Reply:
Thanks, I've tried to apply this change in https://github.com/fwcd/kotlin-language-server/pull/181/commits/0330519a1fe5ec5962d37bf31b83d12acdb5d43c